typing/typecore.ml

```diff
@@ -2223,7 +2274,8 @@ let check_counter_example_pat ~counter_example_args
 
 (* this function is passed to Partial.parmatch
    to type check gadt nonexhaustiveness *)
-let partial_pred ~lev ~splitting_mode ?(explode=0) env expected_ty p =
+let partial_pred ~lev ~splitting_mode ~allow_modules ?(explode=0)
+      env expected_ty p =
```

Comment:
Semantically, `allow_modules` is similar to the strangely named `lev` so it should probably come before `splitting_mode`.
Not that this matters that much.

Reply:
reordered, thanks